### PR TITLE
Safari events attaching and detaching fix

### PR DESCRIPTION
**Pull request by radist2s**
I closed my [previous](https://github.com/akre54/Backbone.NativeView/pull/24) pull request. This is better. Also, a I used <code>apply</code> instead of direct calling because it's better to pass all arguments.

### CODE REVIEW

**Comment by akre54** [comment]
Two things. 1) no need to close a pull request to amend it, just push to the same branch and it'll show up. 2) `.apply` is slow, much slower than calling the function directly, and we know exactly what arguments will be passed so there's no need for `arguments` here.

**Comment by radist2s** [comment]
1) Sorry, I created another PR because I removed wrong commit
2) It's unexpectedly for me. I created [small benchmark](http://jsperf.com/direct-call-vs-apply-extended) to test performance, so yes: apply really slower than direct call but no so much.
I updated the code. Check please.

**Comment by akre54** [comment]
It's mostly a style thing, but thanks for fixing. Squash down to one commit and I'll merge.

File: backbone.nativeview.js [hunk 1]
```diff
@@ -19,10 +19,15 @@
   var ElementProto = (typeof Element !== 'undefined' && Element.prototype) || {};
 
   // Cross-browser event listener shims
-  var elementAddEventListener = ElementProto.addEventListener || function(eventName, listener) {
+  var elementAddEventListener = ElementProto.addEventListener ? function(eventName, listener, useCapture) {
+    return this.addEventListener(eventName, listener, useCapture);
```
Comment (akre54) [inline backbone.nativeview.js]: No need to check for useCapture. It's `false` by default, and we always pass `false` below. We could remove it entirely or just move it to here, I don't have a preference.

**Comment by radist2s** [comment]
I used useCapture in my code earlier, it's a reason why I passed it to arguments.

**Pull request merged**